modules/k6/k6.go

```diff
@@ -36,6 +101,19 @@ func WithTestScript(scriptPath string) testcontainers.CustomizeRequestOption {
 		// add script to the k6 run command
 		req.Cmd = append(req.Cmd, target)
 	}
+	return opt
+}
+
+func WithRemoteTestScript(d RemoteTestFileDescription) testcontainers.CustomizeRequestOption {
```

Comment:
Let's add function comments here too. I'd think about if we need to validate the URL here 🤔 Wdyt?